segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorage.java

```diff
@@ -68,9 +68,7 @@
 import static io.pravega.segmentstore.storage.chunklayer.ChunkStorageMetrics.SLTS_CREATE_LATENCY;
 import static io.pravega.segmentstore.storage.chunklayer.ChunkStorageMetrics.SLTS_DELETE_COUNT;
 import static io.pravega.segmentstore.storage.chunklayer.ChunkStorageMetrics.SLTS_DELETE_LATENCY;
-import static io.pravega.shared.MetricsNames.SLTS_STORAGE_USED_BYTES;
-import static io.pravega.shared.MetricsNames.STORAGE_METADATA_NUM_CHUNKS;
-import static io.pravega.shared.MetricsNames.STORAGE_METADATA_SIZE;
+import static io.pravega.shared.MetricsNames.*;
```

Comment:
I thought we disallow star imports. Does checkstyle like this?

Reply:
Apparently checkstyle didn't complain. Looks like rule doesn't apply for static imports.
But fixed it anyway.